config/firebase.js

```diff
@@ -0,0 +1,9 @@
+export default {
+  option: {
+    apiKey: `AIzaSyBwoBOlp2buF__H36gw7vXWT04QMfnEk34`,
```

Comment:
It's intentional, sure you could hide it but it's used with every front-end req. and can be found anyway. Will remove it later but it's ment to be replace with an own firebase apiKey when cloning the repo.